.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly set up the project linters. Currently, I can only see the test.md file ignored but not the node_modules/.